api/tools/src/main/java/org/pmiops/workbench/tools/CommandLineToolConfig.java

```diff
@@ -54,15 +52,6 @@ WorkbenchConfig workbenchConfig(ConfigDao configDao) {
     return gson.fromJson(config.getConfiguration(), WorkbenchConfig.class);
   }
 
-  /**
-   * Returns the Apache HTTP transport. Compare to CommonConfig which returns the App Engine HTTP
-   * transport.
-   */
-  @Bean
-  HttpTransport httpTransport() {
-    return new ApacheHttpTransport();
-  }
-
```

Comment:
Tested via:
```
./gradlew manageLeonardoRuntimes -PappArgs="['list','https://notebooks.firecloud.org','false','','TABULAR']"
```

Reply:
This failed for me:
![image](https://github.com/all-of-us/workbench/assets/24514630/e7872398-272a-4c09-be2c-bddf91a9f3de)
Is the "active profile" something that I don't have setup correctly on my side?

Reply:
Try running with "-i" to get more logs:
```
./gradlew -i manageLeonardoRuntimes ...
```